person.rb

```diff
@@ -1,25 +1,33 @@
+require_relative './nameable'
+
 class Person
-  att_accessor :id, :name, :age
-  def initialize(name='Unknown', age, parent_permission=false)
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Please initialize the `@rentals` array in the constructor as it will be needed for the many-to-many relationship.
- [ ] Please create a method for adding rentals. The method can take 2 parameters: `book` and `date`. Afterwards, you instantiate a new `Rental` and pass in the `date`, `book`, and `person` as `self` to its constructor.